### DISCUSSION TITLE
[Possible bug?] Audio runs (basically) at 2x speed with crackling on Fedora 40 KDE

### DISCUSSION THREAD

**Discussion by Hexality** [ppy/osu#30611] [Q&A] [answered]
https://github.com/user-attachments/assets/d583b3cc-e596-463a-acea-a22f94821316

With the env_vars provided in another discussion to enforce the quantum in pipewire, the problem persisted.
this problem only happens with osu.

https://github.com/user-attachments/assets/91bfc48c-71b9-43a2-bd66-00e0d4efc0f3

Running:
![image](https://github.com/user-attachments/assets/ea2fbe6d-4466-407f-acc4-dc5dfa1508dc)
with osu!lazer version 2024.1009.1

Logs:
[input.log](https://github.com/user-attachments/files/17738893/1731527314.input.log)
[network.log](https://github.com/user-attachments/files/17738894/1731527314.network.log)
[performance.log](https://github.com/user-attachments/files/17738895/1731527314.performance.log)
[auth.log](https://github.com/user-attachments/files/17738896/1731527314.auth.log)
[runtime.log](https://github.com/user-attachments/files/17738897/1731527314.runtime.log)

**Comment by bdach** [comment]
I personally am not willing to provide any "pipewire" configuration advice. Is something you'll have to debug on your own / with help of someone else who can make any sense of that pipewire nonsense.

**Comment by Hexality** [comment, marked as answer]
fixed, quantum values weren't well setup even on the default pipewire configurations shipped by Fedora, which I had to manually set to 1k for it to properly work.
